lib/travis/build/script/go.rb

```diff
@@ -110,10 +110,10 @@ def uses_make
           
           # see https://golang.org/doc/go1.6#go_command
           def uses_15_vendoring
```

Comment:
I overlooked this in the previous PR, so I apologize for saying this late.

This method should be renamed to `uses_15_vendoring?` to make the intention clearer.